frontend/styles/homepage.scss

```diff
@@ -4,127 +4,97 @@
 
 */
 
-.hero-area {
-  background-color: $blue-10;
-  &__image {
-    width: 225px;
-    @include media-breakpoint-up(lg) {
-      width: 250px;
-    }
-  }
-}
-
-.homepage-process {
-  &::before {
-    // z-index: 0;
-    background: $blue-10;
-    content: "";
-    display: block;
-    position: absolute;
-    bottom: 50%;
-    left: 0;
-    right: 0;
-    top: 0;
-  }
-  // margin-bottom: $spacer * 4;
-  &__wrap {
-    background-color: white;
-    border: 2px solid $blue-20;
-    @include media-breakpoint-up(md) {
-      box-shadow: 7px 7px 0 transparentize($blue-30, .85);
-    }
-
+.hero-illustration {
+  width: 12rem;
+  @include media-breakpoint-up(lg) {
+    width: 16rem;
   }
 }
 
 .homepage-steps {
   position: relative;
   &__wrap {
-    position: relative;
-    z-index: 10;
     display: flex;
     flex-direction: column;
+    position: relative;
     margin-top: $spacer * 1.5;
     margin-bottom: $spacer * 2;
+    z-index: 10;
+
     @include media-breakpoint-up(md) {
       flex-direction: row;
       margin-bottom: $spacer;
     }
   }
+
   &__line {
-    z-index: 5;
     position: absolute;
-    top: 0;
-    left: 19px;
-    width:  1px;
+    top: 1.5rem;
+    left: 1.5rem;
+    width: 1px;
     height: 95%;
+    z-index: 5;
     border-left: 1px dashed $gray-500;
+
     @include media-breakpoint-up(md) {
-      top: 18px;
-      left: 19px; // half of number circle
-      width: 100.5%;
+      width: 100%;
       height: 1px;
       border-left: 0;
       border-bottom: 1px dashed $gray-500;
     }
-    @include media-breakpoint-up(lg) {
-      width: 101.3%;
-      top: 22px;
-    }
   }
 }
 
 .homepage-step {
   display: flex;
   align-items: center;
-  margin-bottom: $spacer * 1.5;  
+  margin-bottom: $spacer * 1.5;
+
   &:last-child {
     margin-bottom: 0;
     background-color: white; // hide dashed line
   }
+
   &__text {
     max-width: 255px;
     font-size: $font-size-sm;
     line-height: $line-height-smaller;
   }
+
   &__number {
     flex: 0 0 auto;
     display: flex;
     align-self: flex-start;
     justify-content: center;
     align-items: center;
-    width: 38px;
-    height: 38px;
+    width: 3rem;
+    height: 3rem;
     border: 1px solid $blue-30;
     border-radius: 50%;
     font-size: $font-size-base * 1.125;
     margin-right: $spacer;
     background-color: white;
-    box-shadow: 2px 3px 0 transparentize($gray-500, .75);
+    box-shadow: $box-shadow-gray-sm;
```

Comment:
Die Variable `box-shadow-gray-sm` wird nicht gefunden. Hast du die in `froide` definiert?